src/utils/Logger.h

```diff
@@ -245,6 +245,20 @@ class Logger {
 	/// allow a set of processes for logging
 	bool set_mpi_output_ranks(int num_nums, int* nums);
 
+	// getRunTime
+	double getRunTime(){
+#ifdef USE_GETTIMEOFDAY
```

Comment:
at some point, this should be updated to use std::chrono.
You can either do this now and remove the old code (both USE_GETTIMEOFDAY + stuff in the if branch) or create an issue for that

Reply:
alternatively use the timer class.

Reply:
3f1cca8